### DISCUSSION TITLE
Include 'since ignore-changes-in' option in baseline configuration

### DISCUSSION THREAD

**Discussion by albepwph** [stryker-mutator/stryker-net#2995] [Q&A]
In order to speed up CI runs in pipelines I want to skip some files in my repository. For this I use the _since ignore-changes-in_ configuration option.
 
As of lately I prefer the _baseline_ config option over _since_. 
Because _since_ and _baseline_ are mutually exclusive I would suggest to also include the _ignore-changes-in_  configuration option inside the _baseline_ config.

I would love to hear if this is possible, and if not, Can you enlighten me on how to exclude files when using the _baseline_ feature.

Thank you in advance!

**Comment by rouke-broersma** [comment]
Baseline and Since are only explicitly mutually exclusive right now. As of now `baseline` implicitly still uses `since` under the hood and ignore-changes-in should still work. Once we have managed to untangle Baseline and Since completely we will have to investigate whether or not an ignore-changes-in configuration is still necessary for Baseline or whether the new diff method would be sufficiently accurate that this option is not necessary. We're not there yet.

If ignore-changes-in does not work for baseline as you would expect, please let me know.

**Comment by albepwph** [comment]
Thanks for your quick response! I appreciate it!

Okay, so you are saying that I can include the ignore-changes-in option inside my baseline config?
Or how would I use this in my config.json files.

Can you please give me an example?

Again thanks!

**Comment by rouke-broersma** [comment]
Sorry I realize that this in between scenario is a bit weird. You should set ignore-changes-in on the since config, but set since disabled.

```json
{
  "stryker-config":
  {
    "since":
    {
      "enabled": false,
      "ignore-changes-in": []
    }
  }
}
```

I think this should work.

**Comment by PetrosK-hue** [comment]
Hello there! Could you elaborate a little more about this? (I want exactly the same thing, speed up process with unwanted files slowing down stryker)
